app/views/pics/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>The feed</h2>
+
+<%= link_to "Post a pic", new_pic_path %>
+
+<div id="the-feed">
```

Comment:
This is interesting on use of divs: http://html5doctor.com/downloads/h5d-sectioning-flowchart.png